standard/expressions.md

```diff
@@ -5542,7 +5659,7 @@ assignment
     ;
 
 assignment_operator
-    : '=' | '+=' | '-=' | '*=' | '/=' | '%=' | '&=' | '|=' | '^=' | '<<='
+    : '' | '+=' | '-=' | '*=' | '/=' | '%=' | '&=' | '|=' | '^=' | '<<='
```

Comment:
This doesn't look right; the new first token appears to be empty!

Reply:
I suspect this is what's causing the validation failure, and my guess is my finger slipped at some point. Thanks, will fix.